manifests/0000_50_olm_07-collect-profiles.networkpolicy.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: collect-profiles
+  namespace: openshift-operator-lifecycle-manager
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    capability.openshift.io/name: "OperatorLifecycleManager"
+    include.release.openshift.io/hypershift: "true"
+spec:
+  podSelector:
+    matchLabels:
+      app: olm-collect-profiles
+  ingress:
+    - ports:
+        - port: 8443
+          protocol: TCP
+      from:
+        - namespaceSelector:
+            matchLabels:
+              name: openshift-operator-lifecycle-manager
+        - podSelector:
+            matchLabels:
+              app: olm-operator
+        - podSelector:
+            matchLabels:
+              app: catalog-operator
+  egress:
```

Comment:
@perdasilva does it need to add 8443 for olm-operator and catalog-operator for egress?

Reply:
we get error 
```console
time="2025-05-20T07:17:09Z" level=info msg="error retrieving pprof profile: Get \"https://olm-operator-metrics:8443/debug/pprof/heap\": dial tcp 172.30.230.187:8443: connect: connection timed out"
time="2025-05-20T07:19:20Z" level=info msg="error retrieving pprof profile: Get \"https://catalog-operator-metrics:8443/debug/pprof/heap\": dial tcp 172.30.196.149:8443: connect: connection timed out"
```
so, need it.

same for the other two network policy.